### PR TITLE
Add stickers to shop and related paypal functionality

### PR DESCRIPTION
**Pull request by QuincyLarson**
I tested this by making a purchase and it worked.

### CODE REVIEW

File: client/less/main.less [hunk 1]
```diff
@@ -1116,6 +1116,78 @@ code {
   }
 }
 
+// diagonal sash
```
Comment (SaintPeter) [inline client/less/main.less]: This code should probably go in a seperate `shop.less` file.  We need to keep our code separated for organizational purposes..

Also, if the classes are shop specific they should probably have a shop specific name.

File: server/views/resources/shop.jade [hunk 2]
```diff
@@ -3,14 +3,15 @@ block content
...
-                        img.img-responsive.img-center(src='http://i.imgur.com/e50zjNy.jpg')
-                        h3 Free Code Camp "function call" logo t-shirt
-                    h4 100% Cotton. Unisex. $20 shipped. 
-                    h4 Only available until February 15.
-                    h5 Are you outside the US? Ships from the EU &thinsp;
-                        a(href='https://teespring.com/free-code-camp-t-shirt-eu-shop' target='_blank') here
-                        | .
\ No newline at end of file
+        .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3.text-center
+            img.img-responsive.img-center(src='http://i.imgur.com/MH1CvwY.jpg')
```
Comment (SaintPeter) [inline server/views/resources/shop.jade]: This image is very confusing.  I found it hard to distinguish one sticker from another.  I think a super simple direct image would be the way to go.

**Comment by SaintPeter** [comment]
This seems to work well aside from my concerns about the image.  I ordered a set.  :smile:

File: server/views/resources/shop.jade (hunk 2 above)
Comment (SaintPeter) [inline server/views/resources/shop.jade]: Also, I found it hard to distinguish the scale of the stickers.  I know they're 2", but it's hard to tell from the image.  They may actually be larger on my screen than in real life.

Banana for scale?

File: client/less/main.less (hunk 1 above)
Comment (QuincyLarson) [inline client/less/main.less]: I ended up not using that CSS and have since removed it.

**Comment by QuincyLarson** [comment]
@SaintPeter I can update the image later - maybe get a good photo from a camper to replace the current one - let's go ahead and get this live sooner rather than later :)

**Comment by SaintPeter** [comment]
Done and Done.

If you get me mine soon, maybe I can take a nice demo image.

**Comment by QuincyLarson** [comment]
@SaintPeter I'll send yours out today :)

**Pull request merged**